pkg/distributor/distributor.go

```diff
@@ -182,8 +182,15 @@ type Distributor struct {
 
 	// partitionsRing is the hash ring holding ingester partitions. It's used when ingest storage is enabled.
 	partitionsRing *ring.PartitionInstanceRing
+
+	// For testing.
```

Comment:
Can you expand this comment a little bit? e.g.
```suggestion
	// For testing functionality that relies on timing without having to sleep in unit tests.
```